### PR TITLE
Fix findbugs

### PR DESCRIPTION
**Pull request by Kathrin-Huber**
Mainly renaming of methods from uppercase to lowercase and translating some of them to english.

### CODE REVIEW

File: Kitodo/src/main/java/de/sub/goobi/export/dms/ExportDms.java [hunk 1]
```diff
@@ -159,7 +159,7 @@ public boolean startExport(Process process, String inZielVerzeichnis, ExportDmsT
     }
 
     /**
-     * Start export.
+     * start export.
```
Comment (stweil) [inline Kitodo/src/main/java/de/sub/goobi/export/dms/ExportDms.java]: Why did you introduce lower case here (and also below)?

File: Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/Process.java [hunk 2]
```diff
@@ -399,27 +399,15 @@ public boolean getContainsUnreachableSteps() {
...
     public int getPropertiesSize() {
-        if (this.getProperties() == null) {
-            return 0;
-        } else {
-            return this.getProperties().size();
-        }
+        return this.getProperties().size();
     }
 
     public int getWorkpiecesSize() {
```
Comment (henning-gerhardt) [inline Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/Process.java]: Why did you leave changes for this method out? This method looks like the other changed methods.

**Comment by stweil** [comment]
> translating some of them to english

s/english/English/. Why did you translate some method names, but changed other German names to lower case without translating them?

By the way, I'd create separate pull requests for FindBugs fixes, removal of unused code and name changes. That also simplifies the review process.

File: Kitodo/src/main/java/de/sub/goobi/export/dms/ExportDms.java (hunk 1 above)
Comment (Kathrin-Huber) [inline Kitodo/src/main/java/de/sub/goobi/export/dms/ExportDms.java]: Just an "automatic-refactoring" error, I didn't see. I already fixed it.

File: Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/Process.java (hunk 2 above)
Comment (Kathrin-Huber) [inline Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/Process.java]: I fixed it.

**Comment by Kathrin-Huber** [comment]
The changes were all made in the context of Findbugs. The main goal was to lower the number of findbugsErrors, so "real" errors (I think, lowercase methods are a minor issue here) do not vanish in the huge amount of findbugs errors.
I translated all methods, where the translation was obvious to me. I just didn't had the time for more translation.

File: Kitodo/src/main/java/de/sub/goobi/forms/ModuleServerForm.java [hunk 3]
```diff
@@ -87,7 +87,7 @@ public void startAllModules() {
     }
 
     /**
-     * Start module server.
+     * start module server.
```
Comment (stweil) [inline Kitodo/src/main/java/de/sub/goobi/forms/ModuleServerForm.java]: Unintended modification?

File: Kitodo/src/main/java/de/sub/goobi/forms/ProjekteForm.java [hunk 4]
```diff
@@ -175,13 +175,13 @@ public String Speichern() {
     }
 
     /**
-     * Apply.
+     * apply.
```
Comment (stweil) [inline Kitodo/src/main/java/de/sub/goobi/forms/ProjekteForm.java]: Unintended modification?

File: Kitodo/src/main/java/de/sub/goobi/helper/CopyFile.java [hunk 5]
```diff
@@ -54,7 +54,7 @@ private static Long createChecksum(SafeFile file) throws IOException {
     }
 
     /**
-     * Start copying of file.
+     * start copying of file.
```
Comment (stweil) [inline Kitodo/src/main/java/de/sub/goobi/helper/CopyFile.java]: Unintended modification?

File: Kitodo/src/main/java/de/sub/goobi/helper/WebDav.java [hunk 6]
```diff
@@ -38,7 +38,7 @@
     private static final Logger myLogger = Logger.getLogger(WebDav.class);
 
     /*
-     * Kopieren bzw. symbolische Links für einen Prozess in das Benutzerhome
+     * copy bzw. symbolische Links für einen Prozess in das Benutzerhome
```
Comment (stweil) [inline Kitodo/src/main/java/de/sub/goobi/helper/WebDav.java]: Unintended modification?

File: Kitodo/src/main/java/de/sub/goobi/helper/ldap/Ldap.java [hunk 7]
```diff
@@ -124,7 +124,7 @@ public boolean isUserPasswordCorrect(User inBenutzer, String inPasswort) {
         myLogger.debug("start login session with ldap");
         Hashtable<String, String> env = getLdapConnectionSettings();
 
-        // Start TLS
+        // start TLS
```
Comment (stweil) [inline Kitodo/src/main/java/de/sub/goobi/helper/ldap/Ldap.java]: Unintended modification?

File: Kitodo/src/main/java/de/sub/goobi/importer/Import.java [hunk 8]
```diff
@@ -51,12 +51,12 @@ public Import() {
     }
 
     /**
-     * Start.
+     * start.
```
Comment (stweil) [inline Kitodo/src/main/java/de/sub/goobi/importer/Import.java]: Unintended modification?

File: Kitodo/src/main/java/de/sub/goobi/metadaten/MetadatenHelper.java [hunk 9]
```diff
@@ -484,7 +484,7 @@ public int compare(final Reference firstObject, final Reference secondObject) {
     /** TODO: Replace it, after Maven is kicked :). */
     private List<Metadata> getAllVisibleMetadataHack(DocStruct inStruct) {
 
-        // Start with the list of all metadata.
+        // start with the list of all metadata.
```
Comment (stweil) [inline Kitodo/src/main/java/de/sub/goobi/metadaten/MetadatenHelper.java]: Unintended modification?

File: Kitodo/src/main/java/de/sub/goobi/metadaten/MetadatenVerifizierung.java [hunk 10]
```diff
@@ -47,7 +47,7 @@
     private final ServiceManager serviceManager = new ServiceManager();
 
     /**
-     * Validate.
+     * validate.
```
Comment (stweil) [inline Kitodo/src/main/java/de/sub/goobi/metadaten/MetadatenVerifizierung.java]: Unintended modification?

File: Kitodo/src/main/java/de/sub/goobi/metadaten/MetadatenVerifizierung.java [hunk 11]
```diff
@@ -69,7 +69,7 @@ public boolean validate(Process process) {
     }
 
     /**
-     * Validate.
+     * validate.
```
Comment (stweil) [inline Kitodo/src/main/java/de/sub/goobi/metadaten/MetadatenVerifizierung.java]: Unintended modification?

File: Kitodo/src/main/java/de/sub/goobi/metadaten/MetadatenVerifizierungWithoutHibernate.java [hunk 12]
```diff
@@ -52,7 +52,7 @@
     private final ServiceManager serviceManager = new ServiceManager();
 
     /**
-     * Validate.
+     * validate.
```
Comment (stweil) [inline Kitodo/src/main/java/de/sub/goobi/metadaten/MetadatenVerifizierungWithoutHibernate.java]: Unintended modification?

File: Kitodo/src/main/java/de/sub/goobi/metadaten/MetadatenVerifizierungWithoutHibernate.java [hunk 13]
```diff
@@ -74,7 +74,7 @@ public boolean validate(Process inProzess) {
     }
 
     /**
-     * Validate.
+     * validate.
```
Comment (stweil) [inline Kitodo/src/main/java/de/sub/goobi/metadaten/MetadatenVerifizierungWithoutHibernate.java]: Unintended modification?

File: Kitodo/src/main/java/de/sub/goobi/metadaten/MetadatenVerifizierungWithoutHibernate.java [hunk 14]
```diff
@@ -566,7 +566,7 @@ public void setAutoSave(boolean autoSave) {
     }
 
     /**
-     * Validate identifier.
+     * validate identifier.
```
Comment (stweil) [inline Kitodo/src/main/java/de/sub/goobi/metadaten/MetadatenVerifizierungWithoutHibernate.java]: Unintended modification?

File: Kitodo/src/main/java/org/goobi/io/BackupFileRotation.java [hunk 15]
```diff
@@ -47,7 +47,7 @@
     private String processDataDirectory;
 
     /**
-     * Start the configured backup.
+     * start the configured backup.
```
Comment (stweil) [inline Kitodo/src/main/java/org/goobi/io/BackupFileRotation.java]: Unintended modification?

File: Kitodo/src/main/java/org/goobi/production/export/ExportDocket.java [hunk 16]
```diff
@@ -79,7 +79,7 @@ public void startExport(Process process, OutputStream os, String xsltfile) throw
     }
 
     /**
-     * Start export.
+     * start export.
```
Comment (stweil) [inline Kitodo/src/main/java/org/goobi/production/export/ExportDocket.java]: Unintended modification?

File: Kitodo/src/main/java/org/goobi/production/export/ExportXmlLog.java [hunk 17]
```diff
@@ -79,7 +79,7 @@ public void startExport(Process p, String destination) throws IOException {
     }
 
     /**
-     * Start export.
+     * start export.
```
Comment (stweil) [inline Kitodo/src/main/java/org/goobi/production/export/ExportXmlLog.java]: Unintended modification?

File: Kitodo/src/main/java/org/kitodo/production/plugin/importer/massimport/googlecode/fascinator/redbox/sru/SRUClient.java [hunk 18]
```diff
@@ -180,7 +180,7 @@ public SRUClient(String baseUrl, String schema, String packing, String version)
             throw ex;
         }
 
-        // Start with the default NLA parameters if nothing has been configured
+        // start with the default NLA parameters if nothing has been configured
```
Comment (stweil) [inline Kitodo/src/main/java/org/kitodo/production/plugin/importer/massimport/googlecode/fascinator/redbox/sru/SRUClient.java]: Unintended modification?

File: Kitodo/src/main/resources/hibernate.cfg.xml [hunk 19]
```diff
@@ -50,7 +50,7 @@
         <property name="show_sql">false</property>
         <property name="hibernate.bytecode.use_reflection_optimizer">false</property>
 
-        <!-- Validate database schema on startup -->
+        <!-- validate database schema on startup -->
```
Comment (stweil) [inline Kitodo/src/main/resources/hibernate.cfg.xml]: Unintended modification?

File: Kitodo/src/main/resources/goobi_config.properties [hunk 20]
```diff
@@ -131,7 +131,7 @@ ApplicationIndividualHeader=<table><tr><tr><td><a style\="color\:white" target\=
 #       used. If this directory exists BUT DOES NOT CONTAIN suitable resources,
 #       the screens will not work as expected.
 localMessages=/usr/local/kitodo/messages/
-# Start-up language: If not set, Kitodo.Production will start up with the
+# start-up language: If not set, Kitodo.Production will start up with the
```
Comment (stweil) [inline Kitodo/src/main/resources/goobi_config.properties]: Unintended modification?

File: Kitodo/src/main/webapp/newpages/ProjekteBearbeiten.jsp [hunk 21]
```diff
@@ -108,7 +108,7 @@
                                                                activeSubStyleClass="activeSub"
                                                                inactiveSubStyleClass="inactiveSub"
                                                                tabContentStyleClass="tabContent">
-                                                <%--<htm:div id="demo" styleClass="yui-navset"> <%--YUI obere Navigation Start --%>
+                                                <%--<htm:div id="demo" styleClass="yui-navset"> <%--YUI obere Navigation start --%>
```
Comment (stweil) [inline Kitodo/src/main/webapp/newpages/ProjekteBearbeiten.jsp]: Unintended modification?

**Comment by Kathrin-Huber** [comment]
Thanks for pointing them all out! I've fixed them.

**Pull request merged**